src/nacl/bindings/crypto_kx.py

```diff
@@ -38,8 +38,9 @@
 def crypto_kx_keypair():
     """
     Generate a keypair.
-    This is a duplicate crypto_box_keypair, but
-    is included for api consistency.
+    This is functionally the same as crypto_box_keypair, however
```

Comment:
Sorry for not having been clear enough; the previous version of the docstring was right. The only docstring needing changes was the one for `crypto_kx_seed_keypair`

Reply:
No worries. My bad.